WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/projectregistration/ProjectRegistrationPage.java

```diff
@@ -139,9 +142,15 @@ public void validateFormInputs() {
 			setErrorMessageAndPageComplete(null);
 		}
 
-		setTitle(TITLE + " (2/3)");
+		setTitle(TITLE + " (" + currentPageNumber + "/"
+				+ ((RegistrationWizard) getWizard()).getTotalPageNumber() + ")");
 		if (shouldSkipProjectSliderPage()) {
-			setTitle(TITLE + " (2/2)");
+			setTitle(TITLE
```

Comment:
By pulling this calculation of the page number up into the base class, you avoid a lot cloned and difficult to read code.